chips/sam4l/src/usart.rs

```diff
@@ -117,7 +117,8 @@ impl USART {
     }
 
     fn set_baud_rate(&self, baud_rate: u32) {
-        let cd = 48000000 / (8 * baud_rate);
+        let system_frequency = 48000000;
```

Comment:
And, sorry, to be clear, I would think the way to do that is to add a function to the `pm` module (or something) that returns the frequency